docs/source/general/how_to_extend_django_freeradius.rst

```diff
@@ -0,0 +1,114 @@
+===============================
+How to extend django-freeradius
+===============================
+
+``django-freeeadius`` provieds set of models, admin and API classes which can be imported, extended and hence customized by third party apps.
+
+Update Settings
+---------------
+
+Update the settings to trigger the swapper:
+
+.. code-block:: python
+
+    # In settings.py of your project
+        
+    DJANGO_FREERADIUS_RADIUSREPLY_MODEL = "my_radius_app.RadiusReply"
+    DJANGO_FREERADIUS_RADIUSGROUPREPLY_MODEL = "my_radius_app.RadiusGroupReply"
+    DJANGO_FREERADIUS_RADIUSCHECK_MODEL = "my_radius_app.RadiusCheck"
+    DJANGO_FREERADIUS_RADIUSGROUPCHECK_MODEL = "my_radius_app.RadiusGroupCheck"
+    DJANGO_FREERADIUS_RADIUSACCOUNTING_MODEL = "my_radius_app.RadiusAccounting"
+    DJANGO_FREERADIUS_NAS_MODEL = "my_radius_app.Nas"
+    DJANGO_FREERADIUS_RADIUSUSERGROUP_MODEL = "my_radius_app.RadiusUserGroup"
+    DJANGO_FREERADIUS_RADIUSPOSTAUTHENTICATION_MODEL = "my_radius_app.RadiusPostAuth"
+
+    # where my_radius_app is name of your app extending django_freeradius
+
+
+Extend models
+-------------
+
+Apart from extending implemented models, ``django_freeradius`` also provides flexibility to extend abstract class models from `django-freeradius.base.models`.
+
+Example:
+
+.. code-block:: python
+
+    # In my_radius_app/models.py
+
+    from django.db import models
+    from django_freeradius.base.models import AbstractRadiusCheck
+    
+    class RadiusCheck(AbstractRadiusCheck):
+        # modify/extend the default behavour here
+        custom_field = models.TextField()
+
+
+Extend admin
+------------
+
+Similar to models, abstract admin classes from ``django_freeradius.base.admin`` can also be extended to avoid duplicate code.
+
+.. code-block:: python
+
+    # In my_radius_app/admin.py
+    
+    from django.contrib import admin 
+    from .models import RadiusCheck
+    from django_freeradius.base.admin import AbstractRadiusAccountingAdmin
+
+    class RadiusCheckAdmin(AbstractRadiusCheckAdmin):
+        model = RadiusCheck
+        # modify/extend default behaviour here
+        fields = AbstractRadiusCheckAdmin.fields + ['custom_field']
+        list_display = AbstractRadiusCheckAdmin.list_display + ['custom_field']
+
+    admin.site.register(RadiusCheck,RadiusCheckAdmin)
```

Comment:
leave a space after the comma (PEP8)

Reply:
As in the previous comment for views, can you suggest this link for a real world example?
https://github.com/openwisp/openwisp-radius/blob/master/openwisp_radius/models.py